test/models/ht_user_test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'byebug'
+
+class HTUserTest < ActiveSupport::TestCase
```

Comment:
I'd like to see some testing of the escaping/unescaping here.

Reply:
Test added